packages/vulkansdk.rb

```diff
@@ -0,0 +1,64 @@
+require 'package'
+
+class Vulkansdk < Package
+  description 'Vulkan SDK'
+  homepage 'https://vulkan.lunarg.com/'
+  version '1.3.280.1'
+  license 'https://vulkan.lunarg.com/license/#/license'
+  compatibility 'x86_64'
+  min_glibc '2.31'
+  source_url 'https://sdk.lunarg.com/sdk/download/1.3.280.1/linux/vulkansdk-linux-x86_64-1.3.280.1.tar.xz'
+  source_sha256 '0bdd5dfc32891eccd38d3377570346040bbdb8921d5294c7ca0f27e0c8ba7332'
+
+  depends_on 'libglvnd'
+  depends_on 'libxcomposite'
+  depends_on 'libxkbcommon'
+  depends_on 'vulkan_icd_loader'
+  depends_on 'xcb_util_image'
+  depends_on 'xcb_util_keysyms'
+  depends_on 'xcb_util_renderutil'
+  depends_on 'xcb_util_wm'
+  depends_on 'xdg_base'
+
+  no_shrink
+  no_compile_needed
+  print_source_bashrc
+
+  def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d"
+    File.write "#{CREW_DEST_PREFIX}/etc/env.d/10-vulkansdk", <<~EOF
+      # Vulkan SDK environment
+      VK_LAYER_PATH=#{CREW_PREFIX}/share/vulkan/explicit_layer.d
+    EOF
+    Dir.chdir 'x86_64' do
+      # Remove conflicts with vulkan_icd_loader
+      FileUtils.rm Dir['lib/cmake/VulkanLoader/*.cmake']
+      FileUtils.rm Dir['lib/libvulkan.so*']
+      FileUtils.mv 'lib', 'lib64'
+      FileUtils.mv Dir['include/vulkan/*'], 'include'
+      FileUtils.rm_rf 'include/vulkan'
+      %w[bin include lib64 share].each do |dir|
```

Comment:
Do we need to worry about `lib` at all here? 

I.e. should that be `%w[bin include #{ARCH_LIB} share].each`...

Reply:
No need since the only compatible architecture is x86_64.